src/preferences/dialog/dlgprefcolors.h

```diff
@@ -17,16 +17,18 @@ class DlgPrefColors : public DlgPreferencePage, public Ui::DlgPrefColorsDlg {
     virtual ~DlgPrefColors();
 
   public slots:
-    // Apply changes to widget
+    /// Called when the preference dialog (not this page) is shown to the user.
```

Comment:
Do we really need to copy documentation from parent classes into every derived class?

Reply:
No, but it doesn't hurt either. I just replaced the already existing comments with better ones.